package.json

```diff
@@ -4,12 +4,13 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build && next export",
```

Comment:
Revert this, make it so that next export command is run every hour with upddated mongodb data